modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -3189,7 +3189,7 @@ optimization. It stays at the center or at a different location specified when C
     @param P2 Output 3x4 projection matrix in the new (rectified) coordinate systems for the second
     camera.
     @param Q Output \f$4 \times 4\f$ disparity-to-depth mapping matrix (see reprojectImageTo3D ).
-    @param flags Operation flags that may be zero or CALIB_ZERO_DISPARITY . If the flag is set,
+    @param flags Operation flags that may be zero or @ref fisheye::CALIB_ZERO_DISPARITY . If the flag is set,
```

Comment:
fisheye has no enum `CALIB_ZERO_DISPARITY` #19089

Reply:
See fisheye enum definition on lines 3005-3016

Reply:
Fisheye test uses cv::CALIB_ZERO_DISPARITY. https://github.com/opencv/opencv/blob/1bfc75ac2337620af59c573339b159886b926057/modules/calib3d/test/test_fisheye.cpp#L404

Reply:
@alalek I propose to add constant to fisheye enum with the same value to presume compatibility and consistence. Whta do you think?

Reply:
need to check that first (constant value should be preserved) through adding of this value.

BTW, this may break some users code which uses both:
- `using namespace cv;`
- `using namespace cv::fisheye;`

AFAIK, there is an idea to unify API for different cameras models, including constants (probably will go to OpenCV 5.x+)

Reply:
`cv::CALIB_ZERO_DISPARITY = 0x00400` (or `1<<10`)
max cv::fisheye enum is `cv::fisheye::CALIB_FIX_PRINCIPAL_POINT = 1 << 9`
therefore, `cv::fisheye::CALIB_ZERO_DISPARITY == cv::CALIB_ZERO_DISPARITY == 1<<10`

Applying the change now.